### DISCUSSION TITLE
Dependabot keeps ignoring minor version

### DISCUSSION THREAD

**Discussion by adri** [community/community#37514] [Code Security]
Hi!

We have previously ignored a minor version upgrade because it tried to upgrade to an RC version by using `@dependabot ignore this minor version`. Now all updates to the minor versions are ignored, we tried to:
* reopen the pull request
* manually update to a (non RC) version, but still within the same minor version. 

The relevant output from a dependabot action is: 
```
93071981> Checking if php 8.1.4-fpm needs updating
updater | INFO <job_493071981> Ignored versions:
updater | INFO <job_493071981>   >= 8.1.a, < 8.2 - from @dependabot ignore command
```

Is there anyway to clear the ignore?

**Comment by ossanna16** [comment]
Hi there @adri and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by adri** [comment]
Never mind we got in touch with support

**Reply by jeffwidman** [reply]
I realize the support request may not be resolved (IDK) but since you transferred the conversation there can you mark this as resolved then? So that when I scan the discussions for "Dependabot" this won't show up in the unresolved ones.
